### ISSUE TITLE
`s3api put-object` failed when specifying to `--key` non-ascii string

### ISSUE THREAD

**Issue by tadashi-aikawa** [localstack/localstack#142]
Hi. Thanks for your great product!
When I used s3api, I found the following error. Is localstack support non-ascii key string?

### OK

```
$ aws --endpoint-url=http://localhost:4572 s3api put-object \
    --bucket mamansoft-jumeaux-viewer \
    --key hoge \
    --body test
{
    "ETag": "\"71a5e7cf60ee860cd14d13aaf5af999f\""
}
```

### NG

```
$ aws --endpoint-url=http://localhost:4572 s3api put-object \
    --bucket mamansoft-jumeaux-viewer \
    --key てすと \
    --body test

An error occurred (500) when calling the PutObject operation (reached max retries: 4): Internal Server Error
```

Best regards.

**Comment by whummer** [comment]
Thanks for reporting @tadashi-aikawa, and apologies for the delay. This should be fixed in 0227d1a . Can you please pull the latest version of the code and/or Docker image (image name `localstack/localstack`, tags `0.7.3` and `latest`) and give it a try? Thanks

**Comment by tadashi-aikawa** [comment]
@whummer 
I confirmed it works as expected. Thank you 😄 

```
$ aws --endpoint-url=http://localhost:4572 s3api put-object \                                                                                                         
    --bucket mamansoft-jumeaux-viewer \
    --key てすと \
    --body test
{
    "ETag": "\"71a5e7cf60ee860cd14d13aaf5af999f\""
}
```

`localstack/localstack:0.7.3`
